edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/IModule.cs

```diff
@@ -112,8 +112,8 @@ public interface IModule : IEquatable<IModule>
         [JsonProperty(PropertyName = "imagePullPolicy")]
         ImagePullPolicy ImagePullPolicy { get; }
 
-        [JsonProperty(PropertyName = "priority")]
-        uint Priority { get; }
+        [JsonProperty(PropertyName = "startupOrder")]
```

Comment:
it looks like the change is not backward compatible, what happens with customers using priority?

Reply:
I don't think this went out for 1.0.9, so there shouldn't be anyone other than internal partners using this yet.

Reply:
https://github.com/Azure/azure-iotedge/releases/
Looks like the feature is first introduced in release 1.0.10-rc1